Plugins/FormatSwift/Plugin.swift

```diff
@@ -20,12 +20,15 @@ struct AirbnbSwiftFormatPlugin: CommandPlugin {
     // specifying the targets selected in the plugin dialog.
     let inputTargets = argumentExtractor.extractOption(named: "target")
 
-    var inputPaths: [String]
+    // Only lint the files passed to `--files`
+    let inputFiles = argumentExtractor.extractOption(named: "files")
+
+    var inputPaths: [String] = inputFiles
     if !inputTargets.isEmpty {
       // If a set of input targets were given, lint/format the directory for each of them
       inputPaths = try context.package.targets(named: inputTargets).map { $0.directory.string }
```

Comment:
Should we support a combination of both `--targets` and `--files`? Right now it looks like if you provide both then we ignore the `--files` inputs. Otherwise if we want to explicitly only support one or the other, I think we should probably throw an error when both are given. Thoughts?

Reply:
Ah yes good catch, I agree! I'll change it to support both.